src/components/ui/post/detail/parts/BoardingPostContent.tsx

```
@@ -0,0 +1,137 @@
+  nearestStation?: string;
+  nearestSchool?: string;
+  gender?: string;
+  isMealProvided?: boolean;
+  options: Array<{ optionId: string; name: string }>;
+  showRoomTour?: boolean;
+  onToggleRoomTour?: () => void;
+  roomTourRef?: React.RefObject<HTMLDivElement>;
+}
+
+export default function BoardingPostContent({
+  isModal,
```

ESLint

[error] 'isModal' is defined but never used.

(@typescript-eslint/no-unused-vars)